PalettePlus/Interop/IpcProvider.cs

```diff
@@ -73,5 +77,9 @@ private static void RemoveCharaPalette(Character chara)
 			return palette.ToString();
 		}
 
+		internal static void PaletteChanged(Character character, Palette palette) {
+			IPaletteChanged?.SendMessage(character, palette.ToString());
```

Comment:
Would it make sense to instead send the entire `Palette` over? (I'd expect that it basically just needs `SerializableAttribute` and it'd work)

Reply:
All the other IPC uses json strings, so I don't see why this one wouldn't do the same

Reply:
I see. I'm not too aware of the Dalamud conventions, sorry.